docs/api/menu-item.md

```diff
@@ -64,6 +64,9 @@ The `role` property can have following values:
 * `zoomin` - Zoom in the focused page by 10%
 * `zoomout` - Zoom out the focused page by 10%
 
+* `menuEdit` - Whole default "Edit" menu (Undo,Copy, etc.)
+* `menuWindow` - Whole default "Window" menu (Minimize, Close, etc.)
```

Comment:
What do you think about calling these `windowMenu` and `editMenu` instead? That reads a little clearer to me.